stl/inc/ranges

```diff
@@ -1333,7 +1333,12 @@ namespace ranges {
 
         constexpr explicit iota_view(_Wi _Value_) noexcept(
             is_nothrow_move_constructible_v<_Wi>&& is_nothrow_default_constructible_v<_Bo>) // strengthened
-            : _Value(_STD move(_Value_)) {}
+            : _Value(_STD move(_Value_)) {
+            if constexpr (totally_ordered_with<_Wi, _Bo>) {
+                _STL_ASSERT(_Value_ <= _Bound, "Per N4950 [range.iota.view]/6, the first argument must precede the "
+                                               "value-initialized bound when their types are totally ordered.");

```

Comment:
Pre-existing: Hmm, I don't think "must precede" correctly describes `<=`. I think this should be "must not exceed".